api/commands/faker.py

```diff
@@ -45,8 +45,8 @@ def fake_events():
 
     @faker_cli.command("groups", help="Fake groups")
     def fake_groups():
-        create_multiple_managers(db.session, 2, 'Group Overseer')
-        create_multiple_managers(db.session, 5, 'Group Leader')
+        # create_multiple_managers(db.session, 2, 'Group Overseer')
+        # create_multiple_managers(db.session, 5, 'Group Leader')
```

Comment:
If we intend to uncomment these lines and use them again, please include a comment to that effect.

If we don't intend to use these lines again, go ahead and just delete them.

Reply:
will be deleted in new commits!